coffee-module/coffee-module-redisstream/src/main/java/hu/icellmobilsoft/coffee/module/redisstream/config/StreamGroupConfig.java

```diff
@@ -19,111 +19,35 @@
  */
 package hu.icellmobilsoft.coffee.module.redisstream.config;
 
-import java.time.Duration;
-import java.util.Optional;
 
-import javax.enterprise.context.Dependent;
 import javax.inject.Inject;
 
 import org.eclipse.microprofile.config.Config;
 
 /**
  * Redis stream group configuration implementation. Key-value par has standard format like yaml file:
- * 
- * <pre>
- * coffee:
- *   redisstream:
- *     enabled: true
- *     sampleGroup:
- *       stream:
- *         maxlen: 10
- *         read:
- *           timeoutmillis: 60000
- *       consumer:
- *         threadsCount: 2
- * </pre>
- * 
+ *
  * @author imre.scheffer
  * @since 1.3.0
  *
  */
-@Dependent
-public class StreamGroupConfig implements IStreamGroupConfig {
+public abstract class StreamGroupConfig {
 
     /**
      * Config delimiter
      */
     public static final String KEY_DELIMITER = ".";
 
-    /**
-     * Prefix for all configs
-     */
-    public static final String REDISSTREAM_PREFIX = "coffee.redisstream";
-
-    /**
-     * Default is no limit. See {@link #getProducerMaxLen()}}
-     */
-    public static final String PRODUCER_MAXLEN = "producer.maxlen";
-
-    /**
-     * Default no ttl, value in millisecond. See {@link #getProducerTTL()}}
-     */
-    public static final String PRODUCER_TTL = "producer.ttl";
-
-    /**
-     * Default 1 minute {@link #getStreamReadTimeoutMillis()}}
-     */
-    public static final String STREAM_READ_TIMEOUTMILLIS = "stream.read.timeoutmillis";
-
-    /**
-     * Default 1 thread {@link #getConsumerThreadsCount()}}
-     */
-    public static final String CONSUMER_THREADS_COUNT = "consumer.threadsCount";
-
-    /**
-     * Default 1 retry count {@link #getRetryCount()}}
-     */
-    public static final String RETRY_COUNT = "consumer.retryCount";
-
-    /**
-     * Default true {@link #isEnabled()}}
-     */
-    public static final String ENABLED = "enabled";
 
     @Inject
-    private Config config;
+    protected Config config;
```

Comment:
Inject-elt mező ne legyen protected, getter vagy a leszármazottba is kerüljön Inject